pom.xml

```diff
@@ -45,6 +45,16 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <dependency>
```

Comment:
We are not having swagger configuration in this pull request
Why did you add these dependencies in this pull request?

Reply:
I have deleted swagger dependency.

Reply:
<artifactId>maven-model</artifactId>
what is the use of this dependency?

Reply:
i think, it is use in swagger .

Reply:
But you have removed swagger dependency, why you did not remove it?